Launcher/default.nix

```diff
@@ -0,0 +1,87 @@
+{ pkgs ? import <nixpkgs> { }
+, stdenv ? pkgs.stdenv
+, lib ? pkgs.lib
+, writeText ? pkgs.writeText
+, dotnet-sdk ? pkgs.dotnet-sdk
+, dotnetPackages ? pkgs.dotnetPackages
+, fetchurl ? pkgs.fetchurl
+, linkFarmFromDrvs ? pkgs.linkFarmFromDrvs
+}:
```

Comment:
Can't we rely on callPackage?

Reply:
This doesn't get called from the flake. If it was, i'd say yes. I haven't managed to build the APPX bundle inside nix (or on linux in general), so this whole file isn't even used in the CI at the moment. We could probably remove this altogether

Reply:
Feel free to give it a shot though, if you want to try to build it